Docker/start-latest.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+cd /opt/data || exit 1
+
+# Download latest version
+curl -L https://ci.appveyor.com/api/buildjobs/51an4dvs204ak6tq/artifacts/MinecraftClient%2Fbin%2FRelease%2FMinecraftClient.exe --output MinecraftClient.exe
```

Comment:
That `51an4dvs204ak6tq` part is randomized by AppVeyor on every new build so getting the latest release would be slightly more involved (request https://ci.appveyor.com/project/ORelio/minecraft-console-client/build/artifacts first to determine the latest download link)

Reply:
Should be fixed. (957dc6e8f882517049be136094b12fd557143da5)

I use the appveyor API and parse the JSON using `jq` to get the latest Job ID  to curl the artifact.